packages/security/.eslintrc.js

```diff
@@ -1,25 +1,31 @@
-// https://eslint.org/docs/user-guide/configuring
-
 module.exports = {
```

Comment:
qua onderhoud is het misschien makkelijker om de .eslintrc naar de root dir te halen

Reply:
ja kan wellicht , maar laten we dat na de bump naar 12 doen, is niet afhankelijk van de node versie